cli-wrap/src/main/resources/cli/da_cli_script.py

```diff
@@ -1,25 +1,59 @@
 #!/usr/bin/env python
 import argparse
 import requests
+requests.packages.urllib3.disable_warnings()
 import sys
 import json
 import os
+import getpass
 
 
-da_server = os.getenv('DA_SERVER', "pnc-da-cli.cloud.pnc.devel.engineering.redhat.com/da/rest/v-0.4")
+with open('config.json') as config_file:    
+    config = json.load(config_file)
+da_server = os.getenv('DA_SERVER', config["daServer"])
 GAV = "GROUP_ID:ARTIFACT_ID:VERSION"
 GA  = "GROUP_ID:ARTIFACT_ID"
 PRODUCT_VERSION = "PRODUCT_NAME:VERSION"
 STATUS_VALUES   = ['SUPPORTED', 'UNSUPPORTED', 'SUPERSEDED', 'UNKNOWN']
 
-def requests_post(link, json_request):
-    return requests_wrapper(requests.post, link, json_request)
+def get_token(login, pswd):
+    server = config["keycloakServer"]
+    client_id = "pncdirect"
+    grant_type="password"
+    params = {'grant_type': grant_type,'client_id': client_id,'username': login,'password': pswd}
+    r = requests.post(server, params, verify=False)
+    if r.status_code == 200:
+        reply = json.loads(r.content.decode('utf-8'))
+        return( str(reply.get('access_token')))
+    else:
+        return ""
+
+def auth():
+    session = requests.Session() 
+    login = raw_input("Enter login (if empty, system name is used):")
```

Comment:
What do you thing about showing the default value? Mabye like this:
>Enter login name [sknot]:

Also is there space/newline printed afte the colon?